Sources/GRPC/GRPCClient.swift

```diff
@@ -29,39 +29,59 @@ extension GRPCClient {
   public func makeUnaryCall<Request: Message, Response: Message>(
     path: String,
     request: Request,
-    callOptions: CallOptions,
+    callOptions: CallOptions? = nil,
```

Comment:
What's the reason for making this optional?

Reply:
The `GRPCClient` protocol also requires `defaultCallOptions`, so we should use those if the `callOptions` aren't specified explicitly. 

If this wasn't the case then if you use `AnyServiceClient` you'd always have to specify `anyClient.make<Type>Call(..., callOptions: anyClient.defaultCallOptions, ...)`